samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -52,6 +52,9 @@
   public static final boolean DEFAULT_DISALLOW_LARGE_MESSAGES = false;
   public static final String DROP_LARGE_MESSAGES = STORE_PREFIX + "%s.drop.large.messages";
   public static final boolean DEFAULT_DROP_LARGE_MESSAGES = false;
+  // The log compaction lag time for transactional state change log
+  public static final String CHANGELOG_MIN_COMPACTION_LAG_MS = STORE_PREFIX + "%s.kafka.changelog.replication.factor";
```

Comment:
Typo in config name (still says replication.factor)

Reply:
Good catch! Let me fix it.